shared/cluster.go

```diff
@@ -373,6 +373,23 @@ func GetPods(print bool) ([]Pod, error) {
 	return pods, nil
 }
 
+// GetPodsByNamespace returns pods parsed from kubectl get pods in a specific namespace
+func GetPodsByNamespace(namespace string, print bool) ([]Pod, error) {
```

Comment:
1) hey lets try to stop adding bool params, only if real needed,they add too much complexity and most of cases not necessary burden as here and the function bellow  for example that u are adding a if print with absolutely no need since u always pass false on the caller

lets let the caller print what they need if they need


2) this function is exact the same as the bellow , so would be better refactor to have only one accepting multiple string and inside your function u check which params you received or is supposed to receive and then u construct your kubectl cmd with the expected vars ( label or not or wahtever other one 

    ) + we already have a kubectlCMD function that does something similiar, if you find best create a separated function like this one "get namespace" its ok, but what is not ok is creating one per argument u might need or not